src/krkn_lib/prometheus/krkn_prometheus.py

```diff
@@ -72,7 +72,7 @@ def process_prom_query_in_range(
         granularity = math.ceil(
             (end_time - start_time).total_seconds() / 11000
         )
-        granularity = granularity if granularity > 0 else 1
+        granularity = granularity if granularity > 10 else 10
```

Comment:
If I remember well this statement was working as a guard because from tests happened that start_time was greater than end_time causing this granularity to be less than 0 (I'm not 100% sure but I'm pretty confident). 

Said that I do agree that this granularity calculation is inconsistent since it will be always 1 (dividing by 11000 will give a 0.something that will be always rounded to 1 unless the chaos run will last more than 11000 seconds that is very unlikely).